datafusion/core/tests/sqllogictests/test_files/dates.slt

```diff
@@ -85,6 +85,6 @@ g
 h
 
 ## Plan error when compare Utf8 and timestamp in where clause
-statement error DataFusion error: Error during planning: The type of Timestamp\(Nanosecond, Some\("\+00:00"\)\) Plus Utf8 of binary physical should be same
+statement error DataFusion error: Error during planning: Timestamp(Nanosecond, Some("+00:00")) + Utf8 can't be evaluated because there isn't a common type to coerce the types to
```

Comment:
```suggestion
statement error DataFusion error: Error during planning: Timestamp\(Nanosecond, Some\("\+00:00"\)\) \+ Utf8 can't be evaluated because there isn't a common type to coerce the types to
```

Regarding https://github.com/apache/arrow-datafusion/pull/5831#issuecomment-1497036876: Sqllogictest supports regex expressions. `()+` are special characters, so the error was parsed as a regex.

Could you please bump the sqllogictest version to `0.13.2` here https://github.com/apache/arrow-datafusion/blob/main/datafusion/core/Cargo.toml#L120. Then `--complete` would work properly for regex-like errors. I am sorry, I somehow missed that the sqllogictest update didn't reach main.

Reply:
Thank you! @melgenek

Reply:
> Then --complete would work properly for regex-like errors. I am sorry, I somehow missed that the sqllogictest update didn't reach main

If you run `cargo update` in your workspace `sqlloictest` will get updated, but @melgenek is right until we increase Cargo.toml it won't happen automatically.